test/Templates.Test/Helpers/TemplateTestBase.cs

```diff
@@ -94,6 +107,26 @@ protected void AssertDirectoryExists(string path, bool shouldExist)
             }
         }
 
+        protected void AssertEmptyMigration(string migration)
+        {
+            var fullPath = Path.Combine(TemplateOutputDir, "Data/Migrations");
+            var file = Directory.EnumerateFiles(fullPath).Where(f => f.EndsWith($"{migration}.cs")).FirstOrDefault();
+
+            Assert.NotNull(file);
+            var contents = File.ReadAllText(file);
+
+            var emptyMigration = @"protected override void Up(MigrationBuilder migrationBuilder)
+        {
+
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+
+        }";
+            Assert.Contains(emptyMigration, contents);
```

Comment:
Don't you have to check other files? Or is the idea that if something fails we'll just regenerate everything under Data/Migrations

Reply:
It's not really required, but we could try to verify the snapshot. But I'm not sure it is worth it since I can't think of a way that it could be wrong without the actual generated migration being non-empty,

Reply:
Yeah, that was my point. I just wanted to confirm

Reply:
Yeah the main idea is just to have tripwire so we have a failure

Reply:
If we ever end up with a situation where something in the migration is wrong and the test didn't fail, we can update the test with that additional case, but I don't think its worth spending too much time making this any more complicated up front